includes/sanitizers/trait-amp-noscript-fallback.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Trait AMP_Noscript_Fallback.
+ *
+ * @package AMP
+ */
+
+/**
+ * Trait AMP_Noscript_Fallback
+ *
+ * @since 1.1
+ *
+ * Used for sanitizers that place <noscript> tags with the original nodes on error.
+ */
+trait AMP_Noscript_Fallback {
+
+	/**
+	 * Attributes allowed on noscript fallback elements.
+	 *
+	 * This is used to prevent duplicated validation errors.
+	 *
+	 * @since 1.1
+	 * @var array
+	 */
+	private $noscript_fallback_allowed_attributes = array();
+
+	/**
+	 * Extends the default arguments array with additional defaults.
+	 *
+	 * @since 1.1
+	 *
+	 * @param array $default_args Original defaults.
+	 * @return array Extended defaults.
+	 */
+	protected function merge_default_args( array $default_args ) {
```

Comment:
Is this the right name? When we have other traits then they would potentially have their own default args to merge, so then this should be named uniquely for this trait.

Reply:
Maybe this method isn't even needed? Maybe better to just have each class that uses it define the default arg like they do for other args?

Reply:
Also, some sanitizers may want different defaults.